### PR TITLE
fix: FI-1931 better custom var src name, docs

### PR DESCRIPTION
**Pull request by defnotrobbie**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ class AwsAlias {
 		 * @see https://www.serverless.com/framework/docs/guides/plugins/custom-variables
 		 */
 		this.configurationVariablesSources = {
-			awsAlias: {
+			dibsServerlessAwsAlias: {
```
Comment (defnotrobbie) [inline index.js]: or maybe `slsPluginAwsAlias`?

Comment (defnotrobbie) [reply index.js]: or `pluginDibsServerlessAwsAlias`?

Comment (defnotrobbie) [reply index.js]: `plugin_serverlessAwsAlias`?

Comment (hellatan) [reply index.js]: i think prefixing with `dibs` is fine. Can we have aliases _only_ in plugins?

**Pull request merged**